oteapi_dlite/utils/utils.py

```diff
@@ -33,22 +39,18 @@
 }
 
 
-def init_session(session):
-    """Initialise session if it is not already initialised."""
+def get_collection(session):
```

Comment:
Not sure this is needed. Should be implemented as part of the Pydantic Configuration datamodel, and populated with the contents of the session

Reply:
The main reason for adding it was to remove the logics below from the different strategies.

However, from your comment above, we could add `collection_id` as an optional keyword argument to allow providing it from the configuration.